apps/vscode/vscode.py

```diff
@@ -47,7 +52,7 @@ def filename():
 
 @ctx.action_class("edit")
 class edit_actions:
-    def find(text = None):
+    def find(text=None):
         if is_mac:
```

Comment:
Not really related to the command server changes but uses of `is_mac` can be moved to the mac_ctx.